### DISCUSSION TITLE
499 reported in nginx reverse proxy

### DISCUSSION THREAD

**Discussion by wizicer** [pocketbase/pocketbase#834] [Q&A] [answered]
<blockquote>
HTTP code 499 usually is seen in NGINX's logs. Being a web server, NGINX 499 is able to identify that the problem is not in the server itself, or the entity which sent the request. HTTP error 499 simply means that the client shut off in the middle of processing the request through the server.
</blockquote>

When requesting home page of admin UI, main index is loaded, but resources not, e.g. `/_/assets/index.e13041a6.js` is failed to load. From log of nginx, it reported 499 and then 504:

```
 - - [24/Oct/2022:07:39:32 +0800] "GET /_/assets/index.e13041a6.js HTTP/1.1" 504 562 "http://xxxxx/_/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36 OPR/82.0.4227.23"
 - - [24/Oct/2022:10:27:06 +0800] "GET /_/assets/index.e13041a6.js HTTP/1.1" 499 0 "http://xxxxx/_/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36 OPR/82.0.4227.23"
```

It works when I requesting the original IP with port.

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure what could cause this. As you quoted, 499 is usually returned for example on client-side timeouts (the 504 also seems to confirm it), but I'm not sure why.

Since it works when accessing it with the IP, then most likely it is misconfiguration of the nginx proxy. Could you post here your nginx config or try the example one from https://pocketbase.io/docs/going-to-production/#deploy-behind-reverse-proxy.

_I'll move the issue to Q&A discussion._

**Reply by wizicer** [reply]
Thank you, sorry for not finding this helpful section, it's some misconfiguration in my nginx proxy, the example works!

**Reply by tinguyen1107** [reply]
Hi @wizicer can you share your problem? I got 499 too :(, and I don't know how to fix it

**Reply by wizicer** [reply]
Hi @tinguyen1107 , I forget the details. However, I remember I just follow the steps in above link exactly and it works then.
